spec/support/feature_helper.rb

```diff
@@ -28,5 +28,11 @@ def on_page(page_title)
     yield
   end
 
+  def select_job_number(full_name:, job_number:)
```

Comment:
nice helper!

Reply:
we have some similar ones in `SnapApplicationFormHelper` if you ever need some clever form methods :)

Reply:
this is good to know